### PR TITLE
Copter: remove more variables if RTL is disabled

### PR DESCRIPTION
**Pull request by tatsuy**
This PR removes RTL_ALT_TYPE parameter and rtl_* variables for RTL_* parameter when RTL is disabled.
I confirmed these with RTL disable option:
 1. no RTL_* parameters exist
 2. This change doesn't change the behavior of ADSB avoidance

### CODE REVIEW

**Comment by rmackay9** [comment]
This looks pretty OK to me.  I haven't checked in much detail however..

File: ArduCopter/avoidance_adsb.cpp [hunk 1]
```diff
@@ -144,6 +144,16 @@ void AP_Avoidance_Copter::set_mode_else_try_RTL_else_LAND(Mode::Number mode)
     }
 }
 
+int16_t AP_Avoidance_Copter::get_altitude_minimum() const
+{
+    int16_t altitude = 0;
```
Comment (khancyr) [inline ArduCopter/avoidance_adsb.cpp:149]: maybe
```suggestion
#if MODE_RTL_ENABLED == ENABLED
    // do not descend if below RTL alt
    return copter.g.rtl_altitude;
else
    return 0;
#endif
```

as we don't really need the variable and we should let the complier optimise this

File: ArduCopter/avoidance_adsb.h [hunk 2]
```diff
@@ -19,6 +19,9 @@ class AP_Avoidance_Copter : public AP_Avoidance {
     // helper function to set modes and always succeed
     void set_mode_else_try_RTL_else_LAND(Mode::Number mode);
 
+    // get the minimum altitude when descend
```
Comment (khancyr) [inline ArduCopter/avoidance_adsb.h:22]: `minimum limit altitude allowed on descend ` maybe ?

**Review by khancyr — APPROVED** [review]
Looks good.

The parameter reordering shouldn't matter.$
the change on asdb avoidance is quite simple

File: ArduCopter/avoidance_adsb.cpp (hunk 1 above)
Comment (tatsuy) [reply ArduCopter/avoidance_adsb.cpp]: Thanks! I fixed it.

File: ArduCopter/avoidance_adsb.h (hunk 2 above)
Comment (tatsuy) [reply ArduCopter/avoidance_adsb.h]: Thank you!

**Review by tridge — APPROVED** [review]

**Comment by rmackay9** [comment]
Merged, thanks!

**Pull request merged**